packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -32,6 +32,7 @@ @implementation RCTViewComponentView {
   BOOL _needsInvalidateLayer;
   BOOL _isJSResponder;
   BOOL _removeClippedSubviews;
+  Cursor _cursor;
```

Comment:
Why isn't this reading directly from props?

Reply:
Thanks! I much prefer this approach. The number of ivars were small and it felt off adding a new one.